packages/sitecore-jss-angular/src/jss-component-factory.service.ts

```diff
@@ -19,14 +19,16 @@ import { RawComponent } from './components/raw.component';
 import { isRawRendering } from './components/rendering';
 
 export interface ComponentFactoryResult {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Not possible to define exact type. In case if try to define `unknown` it's required to have a lot of workarounds for types